src/util/output_channels.ts

```diff
@@ -70,21 +158,46 @@ export async function executeCommandInOutputChannels(command: CliCommand, name:
     // eslint-disable-next-line prefer-const
     startProcess = spawn(cmd.cliCommand, cmd.cliArguments);
     CACHED_CHILDPROCESS.set(name, startProcess);
-    startProcess.stdout.on('data', (chunk) => {
+
+    startProcess.stdout.on('data', async (chunk) => {
+      shadowStdout += chunk;
+      if (shadowStdout.toString().includes('Please provide credentials for image registry')) {
```

Comment:
Better to have those strings as constants

Reply:
I will refactor the code in the new PR.